operations/research-and-development/product/product-management-team-handbook.md

```diff
@@ -114,253 +114,253 @@ Before diving too deeply into docs, processes, and everything else - take this c
+    * Join calls with Sales team
   * Feature requests
-    * Uservoice forums - [https://mattermost.uservoice.com/](https://mattermost.uservoice.com/)  
+    * Uservoice forums - [https://mattermost.uservoice.com/](https://mattermost.uservoice.com/)
     * [Customer request triage](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning.nodetail&quickFilter=97) 
-      * Track down estimates and early dev feasibility discussions on[ Customer Requests](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning&selectedIssue=CR-132)
+      * Track down estimates and early dev feasibility discussions on [Customer Requests](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=17&projectKey=CR&view=planning&selectedIssue=CR-132)
 * Learn about the market
-  * Sales \(Sales - Steve, Lance, Phillip & Solution Architects - Mark, John, Christian\) 
-    * Sales team overview 
-    * Sales team priorities 
-    * What competitors are we up against in sales 
-    * How does Sales team position the product 
-    * What are typical challenges in sales 
-    * Which sales do we win, which do we lose 
+  * Sales \(Regional VPs and Customer Engineers\)
+    * Sales team overview
+    * Sales team priorities
+    * What competitors are we up against
+    * How does Sales team position the product
+    * What are typical challenges for the Sales team
+    * Which sales do we win, which do we lose
     * What are the most common questions on configurations and trials
-  * Marketing \(Lauren- Product Marketing Manager\)
-    * Marketing team overview 
-    * What are the marketing team’s priorities 
-    * What is our current messaging and positioning around Mattermost 
-    * What is needed from product managers
+  * Marketing \(Marketing team lead\)
+    * Marketing team overview
+    * What are the Marketing team’s priorities
+    * What is our current messaging and positioning around Mattermost
+    * What is needed from Product Managers
   * Mini competitive research projects
     * Review of MS Teams, Slack, Discord, and other chat products
-* Spend a week doing customer support 
+* Spend a week doing customer support
   * Review documentation first and work with Support Lead or PM mentor to answer any questions documentation does not address
   * Update docs with answers as appropriate
-* Spend a couple weeks joining as many customer and prospect calls as possible 
-  * Take detailed notes and post to [https://community-daily.mattermost.com/private-core/channels/customer-feedback](https://community-daily.mattermost.com/private-core/channels/customer-feedback)
-* Learn about the product 
+* Spend a couple weeks joining as many customer and prospect calls as possible
+  * Take detailed notes and post to [https://community-daily.mattermost.com/private-core/channels/customer-feedback](https://community.mattermost.com/private-core/channels/customer-feedback)
```

Comment:
Notes should be filed in Salesforce now, Emily Hollinger has the link to the new process

Reply:
@emilyhollinger, would you be open to sharing the link to the process, or details that would work in this context?

Reply:
@justinegeffen here's the link: https://handbook.mattermost.com/operations/customer-success#logging-a-customer-feedback-in-salesforce